rules/false_positives/k8s_dashboard.yara

```diff
@@ -0,0 +1,16 @@
+rule mode_php_js : override {
+  meta:
+    description = "mode-php.js, mode-php_laravel_blade.js"
+    original_severity = "critical"
+    php_executor = "high"
```

Comment:
medium?

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).